tools/pylib/boutdata/mms.py

```diff
@@ -321,7 +321,7 @@ def write(self, nx, ny, output):
 
         """
 
-        MXG = 2
+        MXG = MXG
```

Comment:
You can make the changes and push to the same branch (JensMadsen/adding_mms_flex) and it will automatically be added to this pull request.